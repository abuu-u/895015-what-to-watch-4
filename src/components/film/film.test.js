import React from "react";
import renderer from "react-test-renderer";
import Film from "./film.jsx";

const onFilmClick = () => {};

const onFilmHover = () => {};

const film = {
  id: 1,
  name: `Fantastic Beasts`,
  previewImage: `img/bohemian-rhapsody.jpg`,
};

it(`Render Film`, () => {
  const tree = renderer
    .create(<Film
      film={film}
      onFilmClick={onFilmClick}
      onFilmHover={onFilmHover}
      index={1}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
