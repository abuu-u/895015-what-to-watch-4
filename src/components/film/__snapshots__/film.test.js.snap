// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Film 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  id={1}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
>
  <a
    href="/films1"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts"
        height="175"
        src="img/bohemian-rhapsody.jpg"
        width="280"
      />
    </div>
  </a>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films1"
      onClick={[Function]}
    >
      Fantastic Beasts
    </a>
  </h3>
</article>
`;
